Assets/Scripts/PlayerManager.cs

```diff
@@ -0,0 +1,39 @@
+using Items.Collectables;
+using System.Collections;
+using System.Collections.Generic;
+using System.Net.NetworkInformation;
+using TMPro;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class PlayerManager : MonoBehaviour
+{
+    [SerializeField] private GameObject inventory;
+
+    private int health;
+    private TextMeshProUGUI healthUI;
+
+    
+    void Start()
+    {
+        health = 4;
+        healthUI = inventory.transform.GetChild(2).gameObject.transform.GetChild(1).gameObject.GetComponent<TextMeshProUGUI>();
```

Comment:
same thing here, i'd tag that icon you want to get and get it with the tag, rather than relying on child order